src/Covid19Dashboard/ChartCarousel/index.jsx

```diff
@@ -140,6 +171,22 @@ class ChartCarousel extends PureComponent {
               </p>
             </div>
           }
+          {/* popup when click on a chart */}
+          {
+            this.state.popupChart ?
+              <Popup
+                title={this.state.popupChart.title}
+                onClose={() => this.closePopupChart()}
+              >
+                <ChartCarousel
+                  {...this.props}
+                  chartsConfig={[this.state.popupChart.config]}
+                  isInPopup
+                  popupOnClick={false}
```

Comment:
Why is the value of this property hardcoded to false here? Also -- I think the property could be named better to demonstrate what it toggles. For example, `popupOnClickEnabled` perhaps

Reply:
because i am pulling the properties from the parent with  {...this.props} i need to make sure that i don't enable popups inside of popups
And i need to include  the properties from the parent because sometimes other properties are used for some of the diverse graphs 

I will think more about using a better variable name

Reply:
ended up going with `enablePopupOnClick`